core/src/main/scala/org/apache/spark/SparkThrowableHelper.scala

```diff
@@ -25,6 +25,10 @@ import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.core.`type`.TypeReference
 import com.fasterxml.jackson.databind.json.JsonMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import org.json4s.{JInt, JString}
+import org.json4s.JsonAST.{JArray, JObject}
+import org.json4s.JsonDSL._
+import org.json4s.jackson.JsonMethods.{compact, pretty, render}
```

Comment:
according to https://github.com/apache/spark/pull/36885 , we want to remove json4s eventually. Shall we use jackson here?

Reply:
done